### ISSUE TITLE
GCP self-host installation error

### ISSUE THREAD

**Issue by riseno** [gitpod-io/gitpod#1990]
I tried to install self-host gitpod on GCP, but error shows as
```
Error: googleapi: Error 400: Master version "1.15.12-gke.9" is unsupported., badRequest
```

is that something that i wrong or its the issue in gitpod terraform config?

**Comment by csweichel** [comment]
It would seem that the installer chooses an outdated version of GKE. Did you use the "one line" Docker-based installer or use the terraform scripts directly?

**Comment by kylelemons** [comment]
This reproduces for me with the online docker installer.

The current version for the [stable track](https://cloud.google.com/kubernetes-engine/docs/release-notes) is "1.16.13-gke.401".

To avoid having to keep the version up to date, you can use the terraform [google_container_engine_versions](https://www.terraform.io/docs/providers/google/d/container_engine_versions.html) data provider:

```
data "google_container_engine_versions" "gke" {
  provider       = google-beta
}
...
node_version = data.google_container_engine_versions.gke.release_channel_default_version["STABLE"]
```

**Comment by riseno** [comment]
> It would seem that the installer chooses an outdated version of GKE. Did you use the "one line" Docker-based installer or use the terraform scripts directly?

i ran this command to install on GCP
```
docker run --rm -it \
    -v $PWD/gcloud:/root/.config/gcloud \
    -v $PWD/gpinstall:/workspace \
    eu.gcr.io/gitpod-io/self-hosted/installer:latest \
    gcp
```

**Comment by ranglang** [comment]
same error

**Comment by ranglang** [comment]
googleapi: Error 400: Master version "1.15.12-gke.9" is unsupported., badRequest

```
docker run --rm -it \
    -v $PWD/gcloud:/root/.config/gcloud \
    -v $PWD/gpinstall:/workspace \
    eu.gcr.io/gitpod-io/self-hosted/installer:latest \
    gcp

```

**Comment by cozyGalvinism** [comment]
I have the same issue... Unfortunately, I have no idea how to re-build the image myself ^^'

**Comment by dbanck** [comment]
You don't have to rebuild the image yourself. After running it for the first time, you can edit the terraform files in the `$PWD/gpinstall` folder.

I've updated the `min_master_version` in `gpinstall/gitpod/gcp/modules/kubernetes/main.tf` to match the current GCP version and the installation is working.

**Comment by cozyGalvinism** [comment]
Ah, I missed that. Thanks. I will try that and report back.

**Comment by Flashthedev** [comment]
I managed to change the file but I'm not sure what to do now. What command did you use instead of the docker one as it rebuilds it?

**Comment by dbanck** [comment]
@Flashthedev you should be able to run the same docker command again:

```
docker run --rm -it \
    -v $PWD/gcloud:/root/.config/gcloud \
    -v $PWD/gpinstall:/workspace \
    eu.gcr.io/gitpod-io/self-hosted/installer:latest \
    gcp
```

It will use your updated files from the `$PWD/gpinstall` directory

**Comment by corneliusludmann** [comment]
Should be fixed with #2217.
